docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['autoapi.extension']
+extensions = ['autoapi.extension', 'sphinx.ext.autosummary']
```

Comment:
+1 on moving the addition of this to our extension. We shouldn't force users to have to specify this extension to use autoapi.